### PR TITLE
Add the capability of combining multiple AHI files into one

### PR DESCRIPTION
**Pull request by YaliWu0219**
## Description

Add the capability of combining multiple Himawari8 AHI radiance data files in the ioda format into one. Have tested it in a 4DEnVar experiment in the NCAR/MMM group.

### CODE REVIEW

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]
I think it looks good, but now that you're adding AHI, can you change the name of the script from 'combine_conv' to something else? 'conv' implies conventional only. Maybe just 'combine_files'?

**Comment by YaliWu0219** [comment]
> I think it looks good, but now that you're adding AHI, can you change the name of the script from 'combine_conv' to something else? 'conv' implies conventional only. Maybe just 'combine_files'?

@CoryMartin-NOAA Thank you for your review. I agree, 'combine_files' makes more sense. This script also combines the GeoVals files.

File: src/gsi-ncdiag/combine_files.py [hunk 1]
```diff
@@ -12,10 +12,10 @@
 import datetime as dt
 from pathlib import Path
 
-IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
-if not IODA_CONV_PATH.is_dir():
-    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
-sys.path.append(str(IODA_CONV_PATH.resolve()))
+IODA_FILES_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
```
Comment (CoryMartin-NOAA) [inline src/gsi-ncdiag/combine_files.py:15]: This CONV is for converters not, conventional, need to change these 4 lines back.

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]

File: src/gsi-ncdiag/combine_files.py (hunk 1 above)
Comment (YaliWu0219) [reply src/gsi-ncdiag/combine_files.py]: Oh, I see now.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by srherbener — APPROVED** [review]

**Pull request merged**